climakitae/uncertain_utils.py

```diff
@@ -7,7 +7,7 @@
 import intake
 import warnings
 from .selectors import Boundaries
-from xmip.preprocessing import rename_cmip6
+from cmip6_preprocessing.preprocessing import rename_cmip6
```

Comment:
A common way to deal with changes like this is catching the `ImportError`. This allows for supporting older and newer versions of the package.
``` python
try:
    from xmip.preprocessing import rename_cmip6
except ImportError:
    from cmip6_preprocessing.preprocessing import rename_cmip6
```

Reply:
awesome, just updated